ui/src/pages/__tests__/SearchPage/SearchPageRepos.test.tsx

```diff
@@ -470,9 +484,15 @@ describe("SearchPage component", () => {
 					});
 
 					const scanTimeMatch = screen.getByRole("combobox", {
-						name: /last qualified scan time match/i,
+						name: /last scan time match/i,
```

Comment:
The diff is misleading here. The original `scanTime(Match)` was renamed to `qualifiedScanTime(Match)`, and a new one was created for non-qualified scans (this pattern repeats itself throughout the test files)